inst/uvhydrograph/uvhydrograph.Rmd

```diff
@@ -28,6 +28,7 @@ output:
 ***
 
 ```{r echo=FALSE, warning=FALSE, message=FALSE, results='asis'}
+library(lubridate)
```

Comment:
ugh, sorry. Something must be funky with my environment. Before I merge I'll change that back (again..)